ivy_tests/test_ivy/test_frontends/test_jax/test_array.py

```diff
@@ -2498,6 +2498,48 @@ def test_jax_array_squeeze(
     )
 
 
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="jax.numpy.array",
+    method_name="std",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("float")
```

Comment:
```suggestion
        available_dtypes=helpers.get_dtypes("valid")
```
This should always be `valid`- if the function only supports floating point data then that's what valid will return.